src/StaticCategory/StaticCategory.js

```diff
@@ -1,16 +1,25 @@
 import { categoryIcons } from './categoryIcons.js'
 import StaticItem from '../StaticItem/StaticItem'
+import { BsPlusCircle } from 'react-icons/bs'
 import './StaticCategory.scss'
 
-const StaticCategory = ({ catTitle, items }) => {
+const StaticCategory = ({ catTitle, items, openModal, verifyDeletion }) => {
 
   const generateListItems = () => {
-    return items.map(item => {
+    let sortedItems = items.sort((a, b) => {
+      if(a.name < b.name) { return -1; }
+      if(a.name > b.name) { return 1; }
+      return 0;
+    })
+    return sortedItems.map(item => {
       return (
         <StaticItem
-          key={item} 
+          key={item.item_id} 
           item={item}
+          category={catTitle}
           quantity='Qty.'
```

Comment:
Are we still intending to pass `'Qty.'` as a string right here?

Reply:
Thanks @kncaputo ! We discussed and decided we will leave it for now since we plan to change the styling of the quantity input.